app/src/main/java/edu/cascadia/mobas/campusguidebook/data/dao/ClubDao.java

```diff
@@ -0,0 +1,42 @@
+package edu.cascadia.mobas.campusguidebook.data.dao;
+
+import androidx.lifecycle.LiveData;
+import androidx.room.Delete;
+import androidx.room.Insert;
+import androidx.room.Query;
+import androidx.room.Update;
+
+import java.util.List;
+
+import edu.cascadia.mobas.campusguidebook.data.model.Club;
+import edu.cascadia.mobas.campusguidebook.data.model.User;
+
+
+@androidx.room.Dao
+public interface ClubDao {
+
+    // below method is use to
+    // add data to database.
+    @Insert
+    void insert(Club club);
+
+    // below method is use to update
+    // the data in our database.
+    @Update
+    void update(Club xlub);
```

Comment:
is this supposed to be void update(Club club); ?

Reply:
The magic returning typo. I had already fixed it, but possibly on the wrong branch. This is only used internally by Room, but still needs to be corrected for appearance/consistency.

Reply:
Good catch